profile.py

```diff
@@ -150,7 +151,25 @@ def right_penumbra(self, upper=0.9, lower=0.1):
     def center(self, level=0.5):
         return 0.5 * (self.x_at_y(level) + self.x_at_y(level, reverse=True))
 
-    def mirror(self):
+    def mirror(self, m=0):
+        """
+        Provides mirror image of a profile with given Y axis (y=m).
+        Domain of profile might be changed due to this operation.
+
+        Use mirror() method to get f(-x):
+        >>> foo = LateralProfile([[0, 0], [1, 1], [2, 4], [4, -1]])
+        >>> foo.mirror()
+        >>> print(foo)
```

Comment:
Just a comment: __str__ method will be rewritten in the future. 
From this test you can't see clearly that the profile was mirrored (print(foo) doesn't print data in the object)
Wouldn't look better if you print foo.y ?